chart/kubeapps/templates/kubeappsapis/deployment.yaml

```diff
@@ -92,6 +92,7 @@ spec:
             # TODO (gfichtenolt) this as required by the kubeapps apis service (which will
             # longer-term pass something to the plugins so that the plugins won't need to
             # know these details). Currently they're used directly by the flux plugin
+            {{- if .Values.redis.enabled }}
```

Comment:
Shouldn't that change already be on master from my earlier PR?

Reply:
Yep... perhaps it's an issue when merging (in fact, the `test render` CI step is failing; I'll have a look at it soon)